From 8ff8198e4788dfd4537fa24018473d1d6b0f2702 Mon Sep 17 00:00:00 2001
From: Sailaja <sbandarupalli@habana.ai>
Date: Wed, 13 May 2020 09:25:47 +0300
Subject: [PATCH] [SW-11867] preboot: remove TODO in i2c slave impl

in i2c slave implementation removed hardcoding of temperature sensor
readings. TODO was added as temp sensors readings in preboot was not
implemented

Change-Id: I9f517115d0f249e6e7107d6927afe42d67935ad8
Signed-off-by: Sailaja <sbandarupalli@habana.ai>
---

diff --git a/common/module_i2c_slave.c b/common/module_i2c_slave.c
index 9e4576b..19ad37c 100644
--- a/common/module_i2c_slave.c
+++ b/common/module_i2c_slave.c
@@ -12,6 +12,7 @@
 #include <hl_fw_if.h>
 #include <pcie_dbi_regs.h>
 #include <preboot.h>
+#include <thermal.h>
 
 #include "cmd_module.h"
 #include "uart.h"
@@ -22,9 +23,6 @@
 
 #define HL_DEVICE_STATUS_CMD_CODE_LEN		0x1
 #define HL_PCIE_VENDOR_ID_CMD_CODE_LEN		0x16
-
-/* TODO: Remove this define after actual temp read is implemented */
-#define DUMMY_MAX_TEMP_MC			39000
 
 static unsigned int cmd;
 static unsigned int read_bytes_count;
@@ -185,12 +183,6 @@
 	}
 
 	return ret;
-}
-
-/* TODO; Implement this functionality */
-static int hl_ts_get_max_temp(void)
-{
-	return DUMMY_MAX_TEMP_MC;
 }
 
 static int i2cslave_handle_read_req(void)
